subsys/can/Kconfig

```diff
@@ -0,0 +1,23 @@
+# Kconfig - CAN configuration options
+#
+# Copyright (c) 2019 Erwin Rol <erwin@erwinrol.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig CAN_STACKS
+	bool "CAN support"
```

Comment:
What the STACKS denotes here, is it really needed? Suggestion `menuconfig CANBUS` instead?

Here especially it might be better to say `bool "CANBUS support"` as the CAN word is too generic.